docs/dev/migrating-to-v1.rst

```diff
@@ -165,12 +165,12 @@ reading phase, the data catalog is updated accordingly:
 			name: netcdf
 			filesystem: local
 			metadata_resolver: convention
-			chunks:
-				latitude: 250
-				longitude: 240
-				time: 30
-			combine: by_coords
-			...
+			options:
```

Comment:
`driver_options`? At least I'd like to see some documentation clarifying the difference between `options`, `kwargs`, `driver_kwargs` and any other variation I might have forgotten.

Reply:
I aim to replace driver_kwargs with options under the driver header, I do not like:
```yaml
driver:
  driver_options:
    ...
```

Reply:
ah right, sorry did not properly read the whole context. I think you're right about your example, but I think we still need an internal guideline on how we talk about these because the different kinds is starting to get hard to keep in (human) memory

Reply:
I updated the issue description to make the use case more clear